packages/aft/lib/src/util.dart

```diff
@@ -115,3 +116,34 @@ void sortPackagesTopologically<T>(
         .compareTo(ordered.indexOf(getPubspec(a).name));
   });
 }
+
+// Copyright 2017, the Dart project authors.
+// Copied from io.dart https://pub.dev/documentation/io/latest/io/copyPath.html
```

Comment:
Q: Is this comment/link correct?

Reply:
Discussed offline: Added io package as a dep.